packages/esm-patient-orders-app/src/api/api.ts

```diff
@@ -135,3 +142,48 @@ export function useOrderEncounter(patientUuid: string): {
   }, [isLoadingSystemVisitSetting, errorFetchingSystemVisitSetting, visit, todayEncounter, systemVisitEnabled]);
   return results;
 }
+
+export function useOrderTypes() {
+  const orderTypesUrl = `/ws/rest/v1/ordertype`;
+  const { data, error, isLoading, isValidating } = useSWR<FetchResponse, Error>(orderTypesUrl, openmrsFetch);
+
+  return {
+    data: data?.data?.results,
+    error,
+    isLoading,
+    isValidating,
+  };
+}
+
+export function usePatientOrders(patientUuid: string, status: 'ACTIVE' | 'any', orderType?: string) {
```

Comment:
This is duplicated in [packages/esm-patient-labs-app/src/lab-orders/api.ts](https://github.com/openmrs/openmrs-esm-patient-chart/pull/1463/files#diff-d2930c7ae1ee419fd3e61a7985f5aca6c72931f1addb5d3fd0129d53f7e76395). I believe we can move this to `patient-common-lib`.
Thanks!

Reply:
Agreed. Let me move it